### PR TITLE
Patch fixing stability typo in libgtksourceview-3.0-1.dll's DllMain

### PR DESCRIPTION
**Pull request by galkinvv**
Release of global resources was accidently performed
on any thread exit instead of process exit/dll unload.
Premature releasing caused different stability problems.

This fixes https://github.com/Alexpux/MINGW-packages/issues/3510

### CODE REVIEW

**Comment by galkinvv** [comment]
The master branch in upstream is preparing in to gtk 4 already completely removed the problematic code by introducing explicit init/deinit functions.

So merge to upstream is possible only if 3.24 branch still accepting fixes

File: mingw-w64-gtksourceview3/PKGBUILD [hunk 1]
```diff
@@ -20,13 +20,15 @@ depends=("${MINGW_PACKAGE_PREFIX}-gtk3"
          "${MINGW_PACKAGE_PREFIX}-libxml2")
 options=(!libtool strip staticlibs)
 source=("https://download.gnome.org/sources/gtksourceview/${pkgver%.*}/gtksourceview-${pkgver}.tar.xz"
-        0006-hack-convert-path-back-to-unix.patch)
+        0006-hack-convert-path-back-to-unix.patch 0007-fix-dllmain-typo-thread-to-process.patch)
```
Comment (Alexpux) [inline mingw-w64-gtksourceview3/PKGBUILD]: One patch per line please

**Comment by lazka** [comment]
Nice catch! Please file upstream or prepare an MR :)

File: mingw-w64-gtksourceview3/PKGBUILD [hunk 2]
```diff
@@ -20,13 +20,15 @@ depends=("${MINGW_PACKAGE_PREFIX}-gtk3"
          "${MINGW_PACKAGE_PREFIX}-libxml2")
```
Comment (lazka) [inline mingw-w64-gtksourceview3/PKGBUILD]: Increase pkgrel

**Comment by galkinvv** [comment]
Upstream MR: https://gitlab.gnome.org/GNOME/gtksourceview/merge_requests/3

I fixed issues reported with this PR in second commit (or should I squash commits, like for gnome gitlab MR's - as far as i know)?

**Pull request merged**